SPECS-EXTENDED/rust-coreutils/rust-coreutils.spec

```diff
@@ -0,0 +1,150 @@
+Summary:        Basic system utilities; reimplemented in Rust
+Name:           rust-coreutils
+Version:        0.0.26
+Release:        2%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/uutils/coreutils
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a automatically created tarball with no download link.
+Source1:        rust-coreutils-0.0.26-vendored.tar.gz
```

Comment:
For the vendored tarballs, please create a `generate_source_tarball.sh` script. They are used by the auto-upgrade pipelines to generate new tarballs during an upgrade and they also serve as pointers to how the tarball can be recreated.